src/components/TopBar/TopBarMenu.vue

```diff
@@ -391,6 +391,14 @@ export default {
 		},
 
 		toggleFullscreen() {
+			// Don't toggle fullscreen if there is an open modal
+			if (Array.from(document.body.childNodes).filter(child => {
+				return child.nodeName === 'DIV' && child.classList.contains('modal-mask')
+					&& window.getComputedStyle(child).display !== 'none'
```

Comment:
Looks really non-Vue way because we should tend to query the virtual DOM instead but as modals are now rendered outside Vue hierarchy, this is the shortest way to check their presence.

Reply:
Not sure, but maybe also add a hint why "F" shortkey is not toggling full screen ?

Reply:
Like "showWarning" toast?

Reply:
Exactly